sycl/include/sycl/sub_group.hpp

```diff
@@ -557,7 +557,9 @@ struct sub_group {
   }
 
   /* --- synchronization functions --- */
-  __SYCL_DEPRECATED("Sub-group barrier with no arguments is deprecated.")
+  __SYCL_DEPRECATED(
```

Comment:
There is `barrier(fence_space)` overload below which references this barrier. I think that we should update deprecation message on that overload as well to directly point to a SYCL 2020 replacement